zerver/signals.py

```diff
@@ -0,0 +1,18 @@
+from django.dispatch import receiver
+from django.contrib.auth.signals import user_logged_in
+from django.core.mail import send_mail
+
+import logging
+
+@receiver(user_logged_in)
+def email_on_new_login(sender, user, request, **kwargs):
+	logging.info('New account login. Sending user an email alert.')
+	
+	# email contents
+	subject = "New web login to your Zulip account"
+	message = "Dear User, this is an alert of a new web login into " \
+			  "your account."
```

Comment:
I would write a slightly more involved email for this; check out what other products use for these emails and write something similar.  E.g. the explanation should tell them what to do if it wasn't them (e.g. email `settings.SUPPORT_EMAIL`).

Probably writing a template and putting it under `templates/zerver/emails/`; you can e.g. `git grep followup` to see how to properly template and send emails with more complex content.